From 070c323a7816d9880d69ddc013839c65d7435aae Mon Sep 17 00:00:00 2001
From: Zhantao Tang <zhantao.tang@windriver.com>
Date: Tue, 26 Apr 2022 11:21:06 +0800
Subject: [PATCH] tools: s32ccimage: add reserved member for struct
 program_image to fix build error

The reserved member of struct program_image is added in commit e5d9b759e33f
("secboot: add reserved region to s32gen1image"), but droped in commit
9d247d50979c ("tools: Split s32gen1image.h header"), this will cause the
following build error:

  error: 'struct program_image' has no member named 'reserved'; did you mean 'mbr_reserved'?
  |   235 |  .reserved = {

This patch is to add it back to fix the above error.

Upstream-Status: Pending

Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 tools/s32ccimage.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/s32ccimage.c b/tools/s32ccimage.c
index 0009acc5b1..ec38bbe1d0 100644
--- a/tools/s32ccimage.c
+++ b/tools/s32ccimage.c
@@ -118,6 +118,7 @@ struct program_image {
 	struct image_comp hse_sys_img;
 	struct image_comp app_code;
 	struct image_comp code;
+	struct image_comp reserved;
 	__u8 *header;
 };
 
-- 
2.25.1

